parameter/ParameterMgrFullConnector.cpp

```diff
@@ -40,6 +42,11 @@ CParameterMgrFullConnector::CParameterMgrFullConnector(const string &strConfigur
 {
 }
 
+CommandHandlerInterface *CParameterMgrFullConnector::createCommandHandler()
+{
+    return new CommandHandlerWrapper(_pParameterMgr->createCommandHandler());
```

Comment:
I tried, but as the commandHandler is only movable and the api is c++03, their is no way to avoid this client deletion (other than implementing a `auto_ptr` like semantic that would be more dangerous than useful).